NOTICE

```diff
@@ -0,0 +1,41 @@
+Spark Rapids Tools
+Copyright (c) 2023, NVIDIA CORPORATION
+
+--------------------------------------------------------------------------------
+
+// ------------------------------------------------------------------
+// NOTICE file corresponding to the section 4d of The Apache License,
+// Version 2.0, in this case for
+// ------------------------------------------------------------------
+
+Apache Spark
+Copyright 2014 and onwards The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+--------------------------------------------------------------------------------
+This product bundles various third-party components under other open source licenses.
+
+Boostrap V4.6.1 - The MIT License (MIT)
+    License Text ( https://github.com/twbs/bootstrap/blob/v4.6.1/LICENSE )
+    Copyright (c) 2011-2021 Twitter, Inc.
+    Copyright (c) 2011-2021 The Bootstrap Authors
+
+DataTablesSrc V1.12 - The MIT License (MIT)
+    License Text ( https://datatables.net/license/ )
+    Copyright (C) 2008-present, SpryMedia Ltd.
+
+jQuery V3.6.0 - The MIT License (MIT)
+    License Text ( https://jquery.org/license/ )
+    Copyright OpenJS Foundation and other contributors, https://openjsf.org/
+
+Mustache.js V4.10 - The MIT License (MIT)
+    License Text ( https://github.com/janl/mustache.js/blob/master/LICENSE )
+    Copyright (c) 2009 Chris Wanstrath (Ruby)
+    Copyright (c) 2010-2014 Jan Lehnardt (JavaScript)
+    Copyright (c) 2010-2015 The mustache.js community
+
+Spur Dashboard V1.1.0 - The MIT License (MIT)
+    License Text ( https://github.com/HackerThemes/spur-template/blob/master/LICENSE )
+    Copyright 2016 - 2019 Alexander Rechsteiner
```

Comment:
Do we actually have these sources in our repository, or are they only pulled in at build time into the resulting jar artifact?  If the latter, then these should be in a NOTICE-binary that is bundled in the jar.  IIUC NOTICE is for software that is actually in the repository, not pulled in as a dependency.  cc: @amahussein who may be able to clarify which of these are actually in the source base vs. pulled in and bundled at build time.

Reply:
The sources are only pulled in at build time in the resulting jar.
We should remove them from the NOTICE and have a new NOTICE-binary that contains the section similar to what we had in rapids-tools repo.

Reply:
Updated.